cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
```

Comment:
The `super()` function should be used without specifying the class and instance. Use `super().validate(attrs)` instead of `super(TicketSerializer, self).validate(attrs=attrs)`.